modules/optimizing-energy-use/slides_optimizing_energy_use.md

```diff
@@ -38,7 +38,7 @@ Note:
 
 <!-- .slide: data-state="standard" -->
 
-## What language should I use?
+### What language should I use?
 - Typical conception of energy efficiency:
```

Comment:
would be good to have one of these language comparison plots here, just for illustration.

Reply:
Hmm good idea, have you got one in mind?

Reply:
I've made an issue for it for now (#31)